frontend/frontend-filters.php

```diff
@@ -66,41 +66,47 @@ public function get_locale() {
 	 * @since 0.8
 	 *
 	 * @param int[] $posts List of sticky posts ids.
-	 * @return int[] Modified list of sticky posts ids
+	 * @return int[] Modified list of sticky posts ids.
 	 */
 	public function option_sticky_posts( $posts ) {
 		global $wpdb;
 
-		// Do not filter sticky posts on REST requests as $this->curlang is *not* the 'lang' parameter set in the request
-		if ( ! defined( 'REST_REQUEST' ) && ! empty( $this->curlang ) && ! empty( $posts ) ) {
-			$_posts = wp_cache_get( 'sticky_posts', 'options' ); // This option is usually cached in 'all_options' by WP.
-			$tt_id  = $this->curlang->get_tax_prop( 'language', 'term_taxonomy_id' );
-
-			if ( empty( $_posts ) || ! is_array( $_posts ) || empty( $_posts[ $tt_id ] ) || ! is_array( $_posts[ $tt_id ] ) ) {
-				$posts = array_map( 'intval', $posts );
-				$posts = implode( ',', $posts );
-
-				$languages = array();
-				foreach ( $this->model->get_languages_list() as $language ) {
-					$languages[] = $language->get_tax_prop( 'language', 'term_taxonomy_id' );
-				}
+		// Do not filter sticky posts on REST requests as $this->curlang is *not* the 'lang' parameter set in the request.
+		if ( defined( 'REST_REQUEST' ) || empty( $this->curlang ) || empty( $posts ) ) {
```

Comment:
Is it old code that's never applied?
I mean, can `PLL_Frontent_Filters` be instantiated during a REST request?

Reply:
Possible it's useless now. I prefer to keep as is as it doesn't harm (and don't want to spend time on it now).